pifpaf/drivers/__init__.py

```diff
@@ -74,7 +74,7 @@ def _exec(self, command, stdout=False, ignore_failure=False,
         complete_env = {}
         app = command[0]
 
-        if stdout or wait_for_line:
+        if stdout or wait_for_line or LOG.getEffectiveLevel() >= logging.DEBUG:
```

Comment:
with this file, it will stuck
without it, it will fail as usual, and print the stack trace